cli/cmd/superuser/autoinvite/create.go

```diff
@@ -0,0 +1,53 @@
+package autoinvite
+
+import (
+	"fmt"
+
+	"github.com/rilldata/rill/cli/pkg/cmdutil"
+	"github.com/rilldata/rill/cli/pkg/config"
+	adminv1 "github.com/rilldata/rill/proto/gen/rill/admin/v1"
+	"github.com/spf13/cobra"
+)
+
+func CreateCmd(cfg *config.Config) *cobra.Command {
+	createCmd := &cobra.Command{
+		Use:    "create <org> <domain> <role>",
```

Comment:
nit: I think in other commands we pair `create/delete` and `add/remove` – so either change this to `add` or change the remove command to `delete`